refarch-integrations/refarch-s3-integration/refarch-s3-integration-client-core/src/main/java/de/muenchen/refarch/integration/s3/client/exception/DocumentStorageException.java

```diff
@@ -1,6 +1,10 @@
 package de.muenchen.refarch.integration.s3.client.exception;
 
+import java.io.Serial;
+
 public class DocumentStorageException extends Exception {
+    @Serial
+    private static final long serialVersionUID = 1L;
```

Comment:
We had a simiilar issue in refarch-templates repository. I think it would be better to suppress the warning for missing serial ID here (instead of adding it) as we are not (de)serializing Exception classes.

Reply:
Done